phoebe/atmospheres/passbands.py

```diff
@@ -155,15 +141,14 @@ def __init__(self, ptf=None, pbset='Johnson', pbname='V',
 
         ```py pb.compute_blackbody_intensities() ```
 
-        Step #3: compute Castelli & Kurucz (2004) intensities. To do this, the
-        tables/ck2004 directory needs to be populated with non-filtered
-        intensities available for download from %static%/ck2004.tar.
+        Step #3: instantiate a model atmosphere object and compute intensities:
 
-        ```py atmdir =
-        os.path.abspath(os.path.join(os.path.dirname(os.path.abspath(__file__)),
-        'tables/ck2004')) pb.compute_ck2004_response(atmdir) ```
+        ```py atm = CK2004ModelAtmosphere('path/to/ck2004')
+        pb.compute_intensities(atm) ```
+
+        Step #4: repeat step #3 for other model atmospheres.
 
-        Step #4: -- optional -- import WD tables for comparison. This can only
+        Step #5: -- optional -- import WD tables for comparison. This can only
```

Comment:
do we have a follow-up ticket to update the [passbands tutorial](https://phoebe-project.org/docs/2.4/tutorials/passbands)?